app/avo/resources/maintenance_tasks_run_resource.rb

```diff
@@ -0,0 +1,29 @@
+class MaintenanceTasksRunResource < Avo::BaseResource
+  self.title = :id
+  self.includes = []
+  self.model_class = ::MaintenanceTasks::Run
+  # self.search_query = -> do
+  #   scope.ransack(id_eq: params[:q], m: "or").result(distinct: false)
+  # end
```

Comment:
What is the purpose of `search_query` here?

Reply:
it's commented out in the avo generator template, but it's for making resources searchable